test/suite/test_timestamp18.py

```diff
@@ -127,3 +128,20 @@ def test_ts_writes_with_non_ts_write(self):
                     else:
                         self.assertEqual(cursor[self.get_key(i)], value3)
             self.session.rollback_transaction()
+
+    # check that updates with no timestamp at commit time use first_commit_timestamp
```

Comment:
Comments should start with a capital letter and end with a full stop, even in python tests.

Reply:
Fixed in e2e9065